web/channels/room_channel.ex

```diff
@@ -1,33 +1,24 @@
 defmodule VerkWeb.RoomChannel do
   use Phoenix.Channel
+  alias VerkWeb.TrackingJobsHandler
 
-  @doc """
-  Authorize socket to subscribe and broadcast events on this channel & topic
-
-  Possible Return Values
-
-  `{:ok, socket}` to authorize subscription for channel for requested topic
-
-  `:ignore` to deny subscription/broadcast on this channel
-  for the requested topic
-  """
+  @doc false
   def join("rooms:jobs", _, socket) do
-    Process.flag(:trap_exit, true)
-    send self, :stats
-
+    :ok = GenEvent.add_mon_handler(Verk.EventManager, {TrackingJobsHandler, self}, self)
```

Comment:
So this romm channel will run just when the user goes to `localhost:4000`. The chance that the user will load the first page before Verk is started is preeeetty low and the only issue is that the graphs won't have data being pumped. That's it. No other issues.